docs/pages/enroll-resources/workload-identity/spiffe.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Introduction to SPIFFE
+description: Learn about Secure Production Identity Framework For Everyone (SPIFFE) and how it is implemented by Teleport Workload Identity
+---
+
+SPIFFE (Secure Production Identity Framework For Everyone) is a set of standards
+for securely identifying workloads.
+
+SPIFFE sets out:
+
+- A format for uniquely specifying an identity, the SPIFFE ID.
+- Standards for encoding the SPIFFE ID into verifiable documents, the SVID.
```

Comment:
What does "the SVID" modify here? Is it a rephrasing of "standards"?

Reply:
Ah I'll flip these definitions around to make them more clearly definitions.